Lombiq.Privacy/Handlers/RegistrationFormEventHandler.cs

```diff
@@ -33,7 +33,7 @@ public Task RegistrationValidationAsync(Action<string, string> reportError)
         var registrationCheckbox = _hca.HttpContext?.Request?.Form?[nameof(PrivacyRegistrationConsentCheckboxViewModel.RegistrationCheckbox)]
             .Select(bool.Parse);
 
-        if (registrationCheckbox == null || !registrationCheckbox.Contains(value: true))
+        if (registrationCheckbox.Any() && (registrationCheckbox == null || !registrationCheckbox.Contains(value: true)))
```

Comment:
`registrationCheckbox.Any()` will cause an NRE is `registrationCheckbox` is `null`.